### PR TITLE
IPCP clean-up (stage I)

### PR DESCRIPTION
**Pull request by msune**
Several cleanups and removing the event_loop

Maintainers: @edugrasa @vmaffione

### CODE REVIEW

**Comment by msune** [comment]
Needs testing.

**Comment by vmaffione** [comment]
Have you tested it?

**Comment by msune** [comment]
Yes (Bernat did); I meant reviewers should test it.

File: rinad/src/ipcp/ipc-process.cc [hunk 1]
```diff
@@ -654,372 +663,212 @@ void IPCProcessImpl::processFwdCDAPMsgEvent(
...
+
+		if(!keep_running)
+			break;
+
+		LOG_IPCP_DBG("Got event of type %s and sequence number %u",
+							rina::IPCEvent::eventTypeToString(e->eventType).c_str(),
+							e->sequenceNumber);
+
+		switch(e->eventType){
+			case rina::IPC_PROCESS_DIF_REGISTRATION_NOTIFICATION:
```
Comment (vmaffione) [inline rinad/src/ipcp/ipc-process.cc]: Can we reduce a bit the indentation here, to keep things on 80 cols? I think we can align "switch" and "case" at least.

**Comment by vmaffione** [comment]
Tests are OK. Tested ping and perf on a three node linear topology and three node ring topology.

**Pull request merged**